### ISSUE TITLE
Lint not reported in editor

### ISSUE THREAD

**Issue by jsg2021** [AtomLinter/linter-eslint#1108]
## Issue Type

Bug

## Issue Description

I'm not seeing lint in the editor today. Atom updated to 1.24. I tried switching to the linter ui, thinking it was the atom-ide-ui, or nuclide, but nothing seems to trigger lint in the editor. Running the cli produces the expected results. 

## Bug Checklist
- [x] Restart Atom
- [x] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [x] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Linter Eslint: Debug output here

Atom version: 1.24.0
linter-eslint version: 8.4.1
ESLint version: 4.13.1
Hours since last Atom restart: 0.3
Platform: darwin
Using local project ESLint from: /Users/jgrimes/Workspace/app/node_modules/eslint
Current file's scopes: [
  "source.js.jsx",
  "keyword.control.module.js"
]
linter-eslint configuration: {
  "fixOnSave": true,
  "ignoreFixableRulesWhileTyping": true,
  "lintHtmlFiles": false,
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "disableWhenNoEslintConfig": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDirs": [],
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToSilenceWhileTyping": [],
  "rulesToDisableWhileFixing": []
}
```

**Comment by Arcanemagus** [comment]
Since `atom-ide-ui` provides no debugging functionality can you enable `linter` + `linter-ui-default` and post the results of the `Linter: Debug` command run on a file you are expecting results for?

**Comment by jsg2021** [comment]
```
Platform: darwin
Atom Version: 1.24.0
Linter Version: 2.2.0
Opened file is ignored: No
Matching Linter Providers: 
  - ESLint
Disabled Linter Providers: 
Standard Linter Providers: 
  - ESLint
  - JSON Lint
  - stylelint
Indie Linter Providers: 
  - Java (Eclipse JDT)
  - JavaScript (Flow)
  - JSON (VSCode)
  - PHP (FelixFBecker)
  - Python (pyls)
  - TypeScript (SourceGraph)
UI Providers: 
  - Linter
  - minimap-linter
Ignore Glob: **/*.min.{js,css}
VCS Ignored Paths are excluded: true
Current File Scopes: 
  - *
  - source.js.jsx
  - keyword.control.module.js
```

**Comment by Arcanemagus** [comment]
Hmmmm, is the file modified, or the same state as it is on disk? I notice that you have the `ignoreFixableRulesWhileTyping` option enabled, which would be hiding any messages that ESLint thinks it can fix for you.

**Comment by jsg2021** [comment]
i just turned that on. The file is not modified and produces lint via command line.

**Comment by Arcanemagus** [comment]
Are you able to share the file and ESLint config in question? Or create a minimal repository/gist that reproduces this for you?

**Comment by jsg2021** [comment]
I'm an idiot. .eslintignore had a line "dirname" and it should have been "/dirname" so "/src/dirname" is still linted. :|
